service/src/LevelZero.cpp

```diff
@@ -38,7 +38,7 @@ namespace geopm
                             GEOPM_ERROR_RUNTIME, __FILE__, __LINE__);
         }
 
-        ze_result_t ze_result;
+        ze_result_t ze_result = ZE_RESULT_ERROR_UNKNOWN;
         //Initialize
         ze_result = zeInit(ZE_INIT_FLAG_GPU_ONLY);
```

Comment:
```suggestion
        //Initialize
        ze_result_t ze_result = zeInit(ZE_INIT_FLAG_GPU_ONLY);
```

Reply:
This is preferable, It could also be reasonable to not declare a local variable at all, since the only use in all cases is to pass it to check_ze_result():

```
        check_ze_result(
            zeInit(ZE_INIT_FLAG_GPU_ONLY),
            GEOPM_ERROR_RUNTIME, "LevelZero::" + std::string(__func__) +
            ": LevelZero Driver failed to initialize.", __LINE__);
```

But I'm okay with either solution.